crates/turborepo-repository/src/package_graph/mod.rs

```diff
@@ -91,6 +91,15 @@ pub enum WorkspaceName {
     Other(String),
 }
 
+impl From<WorkspaceName> for String {
```

Comment:
We should prefer implementing `Display` over converting to a string as the former allows us to use string formatting directly without converting to a string: e.g. `println!("The workspace is {}", WorkspaceName::Root)` vs `println!("The workspace is {}", String::from(WorkspaceName::Root))`
```suggestion
impl std::fmt::Display for WorkspaceName {
```

Reply:
Display trait was already implemneted, but turns out i'm not using this anymore so I deleted it